### ISSUE TITLE
Jar incompatability

### ISSUE THREAD

**Issue by vitalyg** [SashaOv/oneclick-hadoop-launcher#1]
It would be great to have a smart system where a version of a jar is bumped, the old jar is deleted from the gateway. Otherwise we will have version incompatibility issues.

**Comment by SashaOv** [comment]
rsync was supposed to take care of that. Do you have test case?

**Comment by vitalyg** [comment]
I am talking about a case that rsync is cannot handle. For example let's say you are using Scalding 0.8.9 and then decide to upgrade to 0.9.0 the plugin will copy the new jar without deleting the old one causing incompatibility issues.

**Comment by SashaOv** [comment]
OK, got it

**Comment by SashaOv** [comment]
I couldn't find out a way to delete redundant files but I can instead specifiy all dependencies explicitly on the classpath, like so: 

```
    HADOOP_CLASSPATH=../lib/slf4j-log4j12-1.4.3.jar:../lib/log4j-1.2.13.jar:../lib/slf4j-api-1.4.3.jar:...
```

Testing...

**Comment by SashaOv** [comment]
Resolved with v0.4
